Makefile

```diff
@@ -2,15 +2,25 @@
 # Configuration Variables #
 ###########################
 # Image URL to use all building/pushing image targets
-export IMAGE_REPO ?= quay.io/operator-framework/operator-controller
-export IMAGE_TAG ?= devel
-export CERT_MGR_VERSION ?= v1.9.0
-export CATALOGD_VERSION ?= $(shell go list -mod=mod -m -f "{{.Version}}" github.com/operator-framework/catalogd)
-export KAPP_VERSION ?= $(shell go list -mod=mod -m -f "{{.Version}}" github.com/vmware-tanzu/carvel-kapp-controller)
-export RUKPAK_VERSION=$(shell go list -mod=mod -m -f "{{.Version}}" github.com/operator-framework/rukpak)
-export WAIT_TIMEOUT ?= 60s
-IMG?=$(IMAGE_REPO):$(IMAGE_TAG)
-TESTDATA_DIR := testdata
+ifeq (, $(IMAGE_REPO))
```

Comment:
These questions are all for the changes to former `?=` statements.  

The gnu recommendation is to have `ifeq(test, expected)`, e.g.

```Makefile
ifeq ($(IMAGE_REPO),)
```

but AFAIK this should be fine.  It should be noted that this will trigger even if the test var is empty, which in this case would overwrite with the default value for the arg.  I can think of no case where we would want the variable to be defined-but-empty, so it's probably fine.
 
Any reason that this is superior to the conditional variable example:

```Makefile
 ifeq ($(origin IMAGE_REPO), undefined)
  IMAGE_REPO = blah
endif
```
?

Finally, this is supposed to be the semantic equivalent of `?=` and it seems reasonable to expect no performance benefit from this particular change.  If that's not the case, how big a performance impact are we seeing?

Reply:
This is not great
```
ifeq ($(origin IMAGE_REPO), undefined)
  IMAGE_REPO = $(shell long-running.sh)
endif
```

This is better
```
ifeq ($(origin IMAGE_REPO), undefined)
  IMAGE_REPO := $(shell long-running.sh)
endif
```

In the first case (which is equivalent to `?=`), `long-running-sh` runs every time `$(IMAGE_REPO)` is referenced.
In the second case, `long-running.sh` only runs once.

Reply:
I'll switch to this style though. I just saw that in the docs as well
```
ifeq ($(origin IMAGE_REPO), undefined)
  IMAGE_REPO := $(shell long-running.sh)
endif
```